common-npm-packages/webdeployment-common-v2/ziputility.ts

```diff
@@ -0,0 +1,78 @@
+import tl = require('azure-pipelines-task-lib/task');
+import path = require('path');
+import Q = require('q');
+import fs = require('fs');
+import { ToolRunner } from 'azure-pipelines-task-lib/toolrunner';
+
+var DecompressZip = require('decompress-zip');
+var archiver = require('archiver');
+
+const deleteDir = (path: string) => tl.exist(path) && tl.rmRF(path);
```

Comment:
This file has been heavily changed. Should we create a separate version for this? Could you please update the descrition also?

Reply:
I have copied this file from master branch and so should be consistent with master branch. My assumption was that master branch will have latest changes.

Reply:
Thank you, I've got it.